src/Http/Controllers/Auth/SsoController.php

```diff
@@ -237,4 +229,18 @@ private function updateMainCharacterId(User $user)
         if (setting('main_character_id') == 0)
             setting(['main_character_id', $user->character_id]);
     }
+
+    /**
+     * If a deactivated user provides a new RefreshToken
+     * the user is reactivated and a note is created.
+     *
+     * @param \Seat\Web\Models\User $user
+     */
+    private function activateUser(User $user)
+    {
+        $user->active = true;
```

Comment:
Is there a missing `save` statement ?
Otherwise, that mean you can create a note but keep the user disabled.

Reply:
No this works just fine 
![image](https://user-images.githubusercontent.com/6583519/49232016-77f5cc80-f3f3-11e8-97b4-07186d564167.png)